package.json

```diff
@@ -32,9 +32,13 @@
     "istanbul": "~0.4.5",
     "jshint": "~2.9.6",
     "mocha": "5.2.0",
+    "chai": "~4.1.2",
     "proxyquire": "0.5.1",
+    "rewire": "^4.0.1",
```

Comment:
Please use ~ instead of ^ and add to CHANGES_NEXT_RELEASE file.

Reply:
Fixed in 71ea6cc